python/pylibcudf/pylibcudf/column.pyx

```diff
@@ -360,6 +377,110 @@ cdef class Column:
             []
         )
 
+    @classmethod
+    def from_ndarray(cls, obj):
```

Comment:
I think we should have two methods:

- `from_array_interface`
- `from_cuda_array_interface`

Or maybe `from_arraylike` where the object supports either the cuda array interface or the array interface?